app/src/common/chromium/com/igalia/wolvic/browser/api/impl/SessionImpl.java

```diff
@@ -50,6 +51,14 @@ private void init() {
         mPanZoomCrontroller = new PanZoomCrontrollerImpl(this);
     }
 
+    private void registerCallbacks() {
+        mTabWebContentsObserver = new TabWebContentsObserver(mRuntime.GetCurrentWebContents(), this);
+    }
+
+    private void unRegisterCallbacks() {
+        mTabWebContentsObserver = null;
+    }
```

Comment:
Since these are used only once, maybe we could call the code directly instead of making them private methods

Reply:
I think we will have more callbacks in `registerCallbacks` later, like mFullScreenCallback, mTabCallback.